hack/test-integration.sh

```diff
@@ -55,7 +55,7 @@ runTests() {
 trap cleanup EXIT
 
 # Convert the CSV to an array of API versions to test
-IFS=',' read -a apiVersions <<< ${KUBE_TEST_API_VERSIONS}
+IFS=',' read -a apiVersions <<< "${KUBE_TEST_API_VERSIONS}"
```

Comment:
I usually avoid IFS because sometimes it has some non-obvious side effects and IMHO it's just an awful interface... But I kind of agree this one case here seems like it should work for all combinations... I don't know, I'd still go for the substitution but I'll let you make the call, pick the one you're more comfortable with.